testing/bdd/templates/active_regions_overlapping.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" ?>
+    <tt:tt ebuttp:sequenceIdentifier="ReSequenced1" 
+    ebuttp:sequenceNumber="12" 
+    ttp:cellResolution="32 15"
+    ttp:timeBase="media"
+    xml:lang="en-GB" 
+    xmlns:ebuttp="urn:ebu:tt:parameters" 
+    xmlns:ebutts="urn:ebu:tt:style" 
+    xmlns:tt="http://www.w3.org/ns/ttml" 
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter"
+    xmlns:tts="http://www.w3.org/ns/ttml#styling" 
+    xmlns:xml="http://www.w3.org/XML/1998/namespace">
+  <tt:head>
+    <tt:styling>
+      <tt:style ebutts:linePadding="0.5c" tts:backgroundColor="rgb(0, 0, 0)" tts:color="rgb(255, 255, 255)" tts:fontFamily="sansSerif" xml:id="SEQ58.defaultStyle1"/>
+      <tt:style ebutts:linePadding="0.5c" tts:color="rgb(255, 255, 255)" tts:fontFamily="sansSerif" xml:id="SEQ59.defaultStyle1"/>
+      <tt:style ebutts:linePadding="0.5c" tts:backgroundColor="rgb(255, 255, 255)" tts:fontFamily="sansSerif" xml:id="SEQ60.defaultStyle1"/>
+      <tt:style ebutts:linePadding="0.5c" tts:backgroundColor="rgb(0, 0, 0)" tts:color="rgb(255, 255, 255)" tts:fontFamily="Serif" xml:id="SEQ61.defaultStyle1"/>
+      <tt:style ebutts:linePadding="0.5c" tts:backgroundColor="rgb(0, 0, 0)" tts:color="rgb(255, 255, 255)" tts:fontFamily="Serif" xml:id="SEQ62.defaultStyle1"/>
```

Comment:
There's some unnecessary stuff here that we should prune out so the test is as simple as it needs to be, i.e. unreferenced styles can be deleted.